src/Microsoft.Framework.PackageManager/Commands/WrapCommand.cs

```diff
@@ -692,7 +692,7 @@ private static string GetNuGetPackagePath(string packagesDir, string id, string
 
         private static string ResolvePackagesDirectory(string projectDir)
         {
-            var rootDir = ProjectResolver.ResolveRootDirectory(projectDir);
```

Comment:
If `ProjectResolver.ResolveRootDirectory` is not used anywhere else, it can be deleted. (I don't know if it is used anywhere else.)